client/my-sites/earn/style.scss

```diff
@@ -19,7 +19,7 @@
 	}
 }
 
-.earn .promo-section__promos .gridicon {
+.earn .promo-section__promos  .action-panel__figure .gridicon {
```

Comment:
Extra specificity is needed here so styling to the main card icon doesn't conflict with the new checkmark or cross icon for feature availability.